### PR TITLE
fixed all bugs

### PR DESCRIPTION
**Pull request by daniilievtukhov**
fixed bugs

### CODE REVIEW

File: src/components/ProfilePage/ProfilePage.jsx [hunk 1]
```diff
@@ -200,6 +221,7 @@ export function ProfilePage() {
                 console.error(error);
             }
         }
+        sessionStorage.setItem("profile", JSON.stringify(profile));
```
Comment (Ruslanchik01) [inline src/components/ProfilePage/ProfilePage.jsx]: No need to give whole 'profile' into session

File: src/components/Layout/AuthModal/components/Register/Register.jsx [hunk 2]
```diff
@@ -7,397 +7,451 @@ import { UserContext } from "../../../../../context/UserContext/UserContext";
...
+                        path: "/",
+                        maxAge: 3600,
+                    });
+                    setCookie("user", JSON.stringify(user), {
+                        path: "/",
+                        maxAge: 3600,
+                    });
+                    setAuth(true);
+                    redirectAfterRegister();
+                    sessionStorage.setItem("profile", JSON.stringify(newUser));
```
Comment (Ruslanchik01) [inline src/components/Layout/AuthModal/components/Register/Register.jsx]: You can change only name and last name

File: src/components/ProfilePage/components/TalentData/TalentData.jsx [hunk 3]
```diff
@@ -15,8 +15,23 @@ import {
...
+        specialization,
+        setSpecialization,
+        talent,
+        setTalent,
+        allTalents,
+        setAllTalents,
+        links,
+        setLinks,
+    } = props;
+    sessionStorage.setItem("profile", JSON.stringify(profile));
```
Comment (Ruslanchik01) [inline src/components/ProfilePage/components/TalentData/TalentData.jsx]: delete this row

**Comment by Ruslanchik01** [comment]
Don't work manual control (change link) in proof pagination

**Comment by Ruslanchik01** [comment]
There are no deleting session after logout
There are no updating session after log in

**Comment by Ruslanchik01** [comment]
Proof block corner not working on hover

**Review by Ruslanchik01 — CHANGES_REQUESTED** [review]

**Review by Ruslanchik01 — COMMENTED** [review]
Required some deleting rows: )

File: src/components/ProfilePage/ProfilePage.jsx [hunk 4]
```diff
@@ -214,10 +214,10 @@ export function ProfilePage() {
...
                         }));
                         setEditting(false);
-                        sessionStorage.setItem("profile", {
-                            first_name: firstName.name,
-                            last_name: lastName.name,
-                        });
+                        // sessionStorage.setItem("profile", {
+                        //     first_name: firstName.name,
+                        //     last_name: lastName.name,
+                        // });
```
Comment (whoisalex1W) [inline src/components/ProfilePage/ProfilePage.jsx:220]: Remove these lines

**Pull request merged**